### DISCUSSION TITLE
Stalled out following `Task [Copying Context]`

### DISCUSSION THREAD

**Discussion by jaredAMD** [microsoft/superbenchmark#304] [Q&A] [answered]
I'm in the process of working through a simple "One managed node, same node as control node" scenario (as specified in the "getting started" documentation) and appear to have stalled out. Specifically, executing the deploy command as follows: 
```bash
sb deploy -f local.ini -i superbench/superbench:v0.4.0-rocm4.2-pytorch1.7
```
will initiate a series of tasks, culminating with `TASK [Copying Context]`, at which point the progress (to the best that I can tell), just holds.  Can you advise? Much appreciated.

**Comment by jaredAMD** [comment]
Not an issue with a **remote** managed node; in this context deploy will progress as anticipated.

**Comment by abuccts** [comment, marked as answer]
Hi, could you try the following command to check Ansible debug log?
```sh
 ANSIBLE_VERBOSITY=4 sb deploy -l localhost -i superbench/superbench:v0.4.0-rocm4.2-pytorch1.7
```
`-l localhost` should do the same as `local.ini`, `ANSIBLE_VERBOSITY=4` will tell which command Ansible is actually running.

Because `TASK [Copying Context]` is the first task that requires `sudo` in `sb deploy`, could you also verify command like `sudo ls` works fine in the local account? If it requires password, pls try with `sb deploy -l localhost --host-password [sudo pass] ...`, although in my environment Ansible always prints error message instead of stuck in this case.
